.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,48 @@
+name: e2e-test
+
+on: [pull_request]
+
+jobs:
+  e2e:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-latest
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          registry-url: 'https://npm.pkg.github.com'
+      - run: echo "registry=https://registry.yarnpkg.com/" > .npmrc
+      - run: echo "@internxt:registry=https://npm.pkg.github.com" >> .npmrc
+      # You cannot read packages from other private repos with GITHUB_TOKEN
+      # You have to use a PAT instead https://github.com/actions/setup-node/issues/49
+      - run: echo //npm.pkg.github.com/:_authToken=${{ secrets.PERSONAL_ACCESS_TOKEN }} >> .npmrc
+      - run: echo "always-auth=true" >> .npmrc
+        env:
+          PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
+
+      - run: yarn
+      - run: touch .env
+      - run: echo "CRYPTO_KEY=6KYQBP847D4ATSFA" >> .env
+      - run: echo "MAGIC_IV=uB5xL72szFpJmjVn2reHN8T2WAVg2yXw" >> .env
+      - run: echo "MAGIC_SALT=qEN1W5R2MNVJKSWx47mZpIkdqwilhazfnhp74f3vEAF05w5j4X0ghfwtJhNmUYDjN7H64VIw91X4ZZYUep1zic9MRCCIiYKVXLkwupvmEhvQv6nbelTBtXn10WVPEYZL" >> .env
+      - run: echo "NEW_CRYPTO_KEY=WZOuIHK0S1vFjjMc" >> .env
+      - run: echo "API_URL=https://desktop.internxt.com" >> .env
+      - run: echo "BRIDGE_URL=https://api.internxt.com" >> .env
+      - run: echo "APP_SEGMENT_KEY=Mf8sJnBfm3HKNt3RpQ8I5PPwjtoeAZK6" >> .env
+      - run: echo "APP_SEGMENT_KEY_TEST=81naPdzAMHb8Sw5HZQb3xSa4yPtE70Op" >> .env
+      - run: echo "BUG_REPORTING_URL=https://desktop-bug-reporting.inxt.workers.dev" >> .env
+      - run: echo "NOTIFICATIONS_URL=https://notifications.internxt.com" >> .env
+      - run: echo "LOCK_REFRESH_INTERVAL=20000" >> .env
+      - run: yarn build
+      - run: xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" -- yarn test:e2e
```

Comment:
In order to run the test on Ubuntu a graphics server is needed

Reply:
It will work, but it needs the `xvfb-run --auto-servernum --server-args="-screen 0 1280x960x24" ` to run on Ubuntu.

Reply:
ah makes sense then